

def DataBaseException():
    print("Access denied for user 'root'@'localhost' (using password: YES)")